purchase_order_hide_receipt_status/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module hides receipt_status field from purchase order form view and tree view to prevent user confusion with reception_status field when purchase_reception_status module installed.
```

Comment:
As a courtesy for better readability.  Although the flake8 setting doesn't check .rst files, it can also be respected here.

```suggestion
This module hides receipt_status field from purchase order form view and tree view to
prevent user confusion with reception_status field when purchase_reception_status module
installed.
```